.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+yarn.lock
```

Comment:
why do you ignore `yarn.lock`?
It's a required file for people who yarn instead of npm.

Reply:
I typically do not include lock files in my repositories because it causes merge hell. The versions in the `package.json` file are sufficient for version control.

Reply:
I think `npm ci` uses the lock file during ci/cd to verify that exact same packages are used every build, not having the lock file could make builds randomly fail when versions are updated.